File: 프로그래머스/lv0/120839. 가위 바위 보/가위 바위 보.java
class Solution {
	public String solution(String rsp) {
		String answer = "";
		String[] arr = rsp.split("");
		for (int i = 0; i < arr.length; i++) {
			if (arr[i].equals("2")) {
				answer += "0";
			}
			if (arr[i].equals("0")) {
				answer += "5";
			}
			if (arr[i].equals("5")) {
				answer += "2";
			}
		}
		return answer;
	}
}

File: 프로그래머스/0/120839. 가위 바위 보/가위 바위 보.kt
class Solution {
    fun solution(rsp: String): String {
       val list = rsp.split("")
        val s = StringBuilder()
        for (i in list) {
            when (i) {
                "2" -> s.append("0")
                "0" -> s.append("5")
                "5" -> s.append("2")
            }
        }

        return s.toString()
    }
}